### PR TITLE
Alerting: remove feature toggle for provisioning API

### PR DESCRIPTION
**Pull request by JohnnyQQQQ**
This PR removes the feature flag for the provisioning API

### CODE REVIEW

**Comment by alexweav** [comment]
Should we also mark the new endpoints as Stable so they will be documented as such? per https://github.com/grafana/grafana/pull/49018

Should just be pasting the tag `stable` into each route definition.

**Comment by JohnnyQQQQ** [comment]
> Should we also mark the new endpoints as Stable so they will be documented as such? per #49018
> 
> Should just be pasting the tag `stable` into each route definition.

Good idea, I marked the routes as stable.

File: pkg/services/ngalert/api/tooling/post.json [hunk 1]
```diff
@@ -813,6 +813,13 @@
      "type": "array",
      "x-go-name": "InhibitRules"
     },
+    "muteTimeProvenances": {
```
Comment (armandgrillet) [inline pkg/services/ngalert/api/tooling/post.json:816]: I believe those changes should have been done in another PR.

**Review by armandgrillet — APPROVED** [review]
Apart from one comment, this makes sense as a revert of https://github.com/grafana/grafana/commit/b8e277ee4c070b64ba4cf024b18c460626d1a2d3
